src/order/order.service.spec.ts

```
@@ -0,0 +1,297 @@
+        {
+          provide: PrismaService,
+          useValue: mockPrismaService,
+        },
+      ],
+    }).compile();
+
+    service = module.get<OrderService>(OrderService);
+    prisma = module.get(PrismaService) as MockPrismaService;
+
+    // Assign the mocked functions
+    mockedCalculateTotalAmount = jest.requireMock('./order.mapper').calculateTotalAmount;
```

ESLint

[error] Insert `⏎·····`

(prettier/prettier)